src/olympia/files/models.py

```diff
@@ -372,6 +379,17 @@ def get_localepicker(self):
         statsd.timing('files.extract.localepicker', (end * 1000))
         return res
 
+    @amo.cached_property
+    def webext_permissions(self):
+        return {name: WEBEXT_PERMISSIONS.get(name, Permission(name, '', ''))
+                for name in json.loads(self.webext_permissions_json)}
```

Comment:
Similarly, with a `JSONField` the `json.loads()` would be done behind the scenes for you.